app/src/main/java/com/yagni/model/MarketRepo.java

```diff
@@ -7,146 +7,158 @@
+            /******** FOR DAILY REPORT ********/
+            ResultSet result = state.executeQuery(
+                    "SELECT product_id, sum(product_quanity) AS 'sum' FROM yagni_inv_db.purchase_history WHERE order_date > DATE_ADD(curdate(),INTERVAL -1 DAY) AND order_date <= curdate() GROUP BY product_id ORDER BY sum(product_quanity) DESC LIMIT 3;");
             result.next();
-            firstProduct = result.getString(1);      //1st "most popular order"
+            firstProduct = result.getString(1);         // 1st "most popular order"
             result.next();
-            secondProduct= result.getString(1);      //2nd "most popular order"
+            secondProduct = result.getString(1);        // 2nd "most popular order"
             result.next();
-            thirdProduct = result.getString(1);      //3rd "most popular order"
+            thirdProduct = result.getString(1);         // 3rd "most popular order"
 
-            result = state.executeQuery("SELECT (SELECT sum(product_quanity) FROM purchase_history) * product.Sale_cost AS 'totalSpent', customer_email FROM product RIGHT JOIN purchase_history ON product.Sale_cost = purchase_history.product_quanity LEFT JOIN  yagni_inv_db.customer_information ON purchase_history.hashed_email = customer_information.hashed_email WHERE  order_date > date_add(curdate(), INTERVAL - 1 DAY) AND order_date <= curdate() group by customer_email ORDER BY totalSpent DESC LIMIT 3;");
+            result = state.executeQuery(
+                    "SELECT (SELECT sum(product_quanity) FROM purchase_history) * product.Sale_cost AS 'totalSpent', customer_email FROM product RIGHT JOIN purchase_history ON product.Sale_cost = purchase_history.product_quanity LEFT JOIN  yagni_inv_db.customer_information ON purchase_history.hashed_email = customer_information.hashed_email WHERE  order_date > date_add(curdate(), INTERVAL - 1 DAY) AND order_date <= curdate() group by customer_email ORDER BY totalSpent DESC LIMIT 3;");
             result.next();
-            firstCustomer= result.getString(2);      //1st "most popular customer"
+            firstCustomer = result.getString(2);        // 1st "most popular customer"
             firstAmt = result.getString(1);
             result.next();
 
-            secondCustomer= result.getString(2);      //2nd "most popular customer"
+            secondCustomer = result.getString(2);       // 2nd "most popular customer"
             secAmt = result.getString(1);
             result.next();
-            
-            thirdCustomer= result.getString(2);        //3rd "most popular customer"
+
+            thirdCustomer = result.getString(2);        // 3rd "most popular customer"
             thirdAmt = result.getString(1);
```

Comment:
Same as my note above about inline comments- can probably move this to one block comment explaining the set.